src/compiler/utilities.ts

```diff
@@ -1134,6 +1140,48 @@ module ts {
     }
 
     // @internal
+    export function nodeStartsNewLexicalEnvironment(n: Node): boolean {
+        return isAnyFunction(n) || n.kind === SyntaxKind.ModuleDeclaration || n.kind === SyntaxKind.SourceFile;
+    }
+
+    // @internal

```

Comment:
Why do we need all these internal annotations? I thought everything in utilities is automatically internal as a result of the way we build